src/main/kotlin/com/manimdsl/errorhandling/ErrorHandler.kt

```diff
@@ -18,6 +18,12 @@ object ErrorHandler {
         semanticErrors.add("Semantic error at $linePos: $errorEvent")
     }
 
+    fun addRuntimeError(errorEvent: String, lineNumber: Int) {
+        println("Error detected during program execution. Animation could not be generated")
```

Comment:
Last thing I think - do we want to be printing these immediately instead of adding to an arraylist like we do for the other errors?